org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -195,6 +210,14 @@ protected IStatus run(IProgressMonitor monitor) {
 				return Status.OK_STATUS;
 			}
 
+			/* (non-Javadoc)
+			 * @see org.eclipse.core.runtime.jobs.Job#belongsTo(java.lang.Object)
+			 */
+			@Override
+			public boolean belongsTo(Object family) {
+				return super.belongsTo(INITIALIZE_AFTER_JOB);
```

Comment:
What does this actually do ? Seems like the parent (Job) just returns `false`.

Reply:
It was a mistake. I have fixed it.